CL/cl_gl_ext.h

```diff
@@ -30,7 +30,7 @@ extern "C" {
 
 extern CL_API_ENTRY cl_event CL_API_CALL
 clCreateEventFromGLsyncKHR(cl_context context,
-                           cl_GLsync  cl_GLsync,
+                           cl_GLsync  /*cl_GLsync*/,
```

Comment:
Minor: I'd prefer to use a different name for this parameter vs. commenting it out.  Maybe `sync`, which is used in the [function description](https://bashbaug.github.io/OpenCL-Docs/html/OpenCL_Ext.html#cl_khr_gl_event-new-procedures-and-functions) in the extension spec?

This comment can be addressed later if needed and shouldn't hold up this PR.

Reply:
No biggie. Changed.